data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java

```diff
@@ -586,4 +587,40 @@ public boolean matches(Object o) {
         };
     }
 
+    @Test
+    public void shouldSendEventsToSuccessTopicForLogEvents() throws Exception {
+        stub(envelopeMock.getMessage()).toReturn(EventFixture.LOG_EVENT);
```

Comment:
What are we testing here?

Reply:
Pushing sample LOG event and checking its output values after denormalization.